src/themes/dark-theme/dark-theme-with-move.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import deepMerge from '../../utils/deep-merge.js';
+import {DarkTheme} from './dark-theme.js';
+import {
+  typography as moveTypography,
+  fontTokens as moveFontTokens,
+} from '../move-theme/typography.js';
+import getTypography from '../shared/typography.js';
+
+import type {ThemeT} from '../../styles/types.js';
+
+export const DarkThemeMove: ThemeT = deepMerge(
+  {
+    name: 'dark-theme-with-move',
+    typography: deepMerge(getTypography(moveFontTokens), moveTypography),
```

Comment:
Same call out as before. This pattern with typography seems a little confusing to understand, but it might be the best options for now to reduce redundancy.

Reply:
It's due to the font family being included in primitives and used to make the typography overrides much simpler rather than passing the whole `theme.typograhy` object. I'll add some comments to clarify it.